hr_attendance_autoclose/models/hr_attendance.py

```diff
@@ -10,6 +10,7 @@ class HrAttendance(models.Model):
     _inherit = "hr.attendance"
 
     @api.multi
+    @api.depends('check_out')
```

Comment:
and `check_in`?

Reply:
doesn't depend of check_in

Reply:
`attendance.check_out - attendance.check_in`?

Reply:
@mreficent ^

Reply:
I think `check_in` is necessary

Reply:
all of you are right, the check_in is necessary.